Slider/Slider.less

```diff
@@ -86,52 +82,58 @@
 				display: block;
 			}
 		}
-
-		&.pressed,
-		&:focus.activateOnFocus,
-		&:focus.active {
-			.knob::before {
-				transform: translate3d(-50%, -50%, 0) scale(1);
-			}
-		}
 	});
 
 	.applySkins({
 		.progressBar {
 			background-color: @agate-slider-bg-color;
+			background-image: @agate-slider-bg-image;
 		}
 
 		.fill {
-			background-color: @agate-slider-bg-image;
+			background-color: @agate-slider-fill-bg-color;
 		}
 
 		.knob::before {
+			// Vertically and horizontally center this knob.
+			// Using this method rather than -top -left so the knob is more intuitively customizable.
+			transform: translate3d(-50%, -50%, 0) scale(0.8) skewX(@agate-slider-knob-skew);
```

Comment:
This whole transform value should probably be in a theme variable, so a theme can customize how/whether the knob is centered/positioned.